server/elastic/src/adapter.ts

```diff
@@ -104,27 +104,23 @@ class ElasticAdapter implements FullTextAdapter {
       if (field !== undefined) {
         console.log('Mapping', mappings.body)
       }
-      let wsMappings = mappings.body.storage_index
-      if (Object.keys(wsMappings?.mappings?.properties ?? {}).some((k) => k.includes('->'))) {
-        await this.client.indices.delete({
-          index: indexName
-        })
-        const createIndex = await this.client.indices.create({
-          index: indexName
-        })
-        console.log('recreate index', createIndex)
-        const mappings = await this.client.indices.getMapping({
-          index: indexName
-        })
-        wsMappings = mappings.body.storage_index
-      }
+      const wsMappings = mappings.body.storage_index
```

Comment:
My mistake, forgot to previously fix name. `mappings.body[indexName]`

Reply:
good catch